data/saas/employees_hiring-agent.yaml

```
@@ -0,0 +1,65 @@
+      status: todo
+    - title: 候補者DB/同意管理
+      status: todo
+      
+  guidelines:
+    editing_files:
+      - data/saas/prompts-repo.yaml: "カード並び順/動画割当/チームダッシュボードに採用担当を追記"
+      - static/data/prompts/hiring-agent.md: "本ファイル（採用担当のプロンプト本文）"
+      - static/videos/recruiter_card_video_square.mp4: "仮動画（差し替え予定）"
+      - static/images/agent_card.png: "任意: ポスター画像"
+      - themes/kamui-docs/static/js/main.js: "DevTools/右クリックInspector（必要に応じて）"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)